### ISSUE TITLE
circular import error

### ISSUE THREAD

**Issue by m-bain** [fgnt/meeteval#28]
Hi,

Thank you for open sourcing these multispeaker ASR metrics :).

Just reporting some issue I encountered:

Installing as follows works fine
```
pip install cython
git clone git@github.com:fgnt/meeteval.git
pip install -e ./meeteval[cli]
```

However installing like:
```
pip install cython
pip install https://github.com/fgnt/meeteval/archive/refs/heads/main.zip
```
Causes circular import issue:
```python
import meeteval
```
> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/lib/python3.7/site-packages/meeteval/__init__.py", line 1, in <module>
    from . import io
ImportError: cannot import name 'io' from 'meeteval' (/opt/conda/lib/python3.7/site-packages/meeteval/__init__.py)

Very strange, wondering if you have any ideas for workarounds for this? The build software we are using doesn't support installation via the first method.

**Comment by boeddeker** [comment]
Thanks for reporting.
I am fixing this in #29.

> The build software we are using doesn't support installation via the first method.

My guess is, that your build environment hasn't setup an ssh key for github.
I changed in the Readme the `git clone` to `git clone https://github.com/fgnt/meeteval`.
The new instructions will most likely work (https doesn't need an authentication for pull).

> Causes circular import issue

The issue was, that the files were missing. In the setup.py we hard-coded the packages to `meeteval`, instead of using find_modules to find also the submodules.
Once I merged my PR, it will be fixed and you could install directly from github.

Additionally, I added tests, that this will also work in the future.

**Comment by boeddeker** [comment]
I merged #29. When I follow your instructions, it is now working. Let me know if the fix is also working for you.

**Comment by m-bain** [comment]
Thank you! This solved it :)
